docs/manual/docs/cmd/login.md

```diff
@@ -68,4 +68,16 @@ Log in to Office 365 using a personal information exchange (.pfx) file
 
 ```sh
 login --authType certificate --certificateFile /Users/user/dev/localhost.pfx --thumbprint 47C4885736C624E90491F32B98855AA8A7562AF1 --password 'pass@word1'
+```
+
+Log in to Office 365 using a system assigned managed identity. Applies to Azure Virtual Machines and Azure Functions with enabled managed identity
+
+```sh
+login --authType identity
+```
+
+Log in to Office 365 using a user-assigned managed identity. Client id or principal id also known as object id value can be specified in the userName option. Applies to Azure Virtual Machines and Azure Functions with enabled managed identity 
```

Comment:
Other Azure resources besided VM & Functions support Managed Identity. Suggest wording similar to: "Applies to Azure resources with managed identity enabled, such as Azure Virtual Machines, Azure App Service or Azure Functions

Reply:
@pschaeflein , I added it that way since I tested it only the VMs and Azure Functions. In theory it should work with Azure Web Apps (since it works with Azure Functions), but I wanted to be more specific until we test it with the other resources. What do you think?

Reply:
Up to you. Just my thoughts.